### PR TITLE
spelling mistake

### PR DESCRIPTION
**Pull request by lunhuijie**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
**Which issue(s) this PR fixes**:

Fixes #
**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
"NONE"
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
```

### CODE REVIEW

File: clusterloader2/cmd/clusterloader.go [hunk 1]
```diff
@@ -334,7 +334,7 @@ func main() {
 		}
 		testConfig, errList := test.CompileTestConfig(ctx)
 		if !errList.IsEmpty() {
-			klog.Exitf("Test compliation failed: %s", errList.String())
+			klog.Exitf("Test compilation failed: %s", errList.String())
```
Comment (yangjunmyfm192085) [inline clusterloader2/cmd/clusterloader.go:337]: Are there similar errors in other locations?

Comment (lunhuijie) [reply clusterloader2/cmd/clusterloader.go:337]: NO

**Pull request merged**